src/spec/integration/importArchive.spec.coffee

```diff
@@ -0,0 +1,174 @@
+Promise = require 'bluebird'
+_ = require 'underscore'
+archiver = require 'archiver'
+_.mixin require('underscore-mixins')
+{Import} = require '../../lib/main'
+Config = require '../../config'
+TestHelpers = require './testhelpers'
+Excel = require 'exceljs'
+cuid = require 'cuid'
+path = require 'path'
+tmp = require 'tmp'
+fs = Promise.promisifyAll require('fs')
+# will clean temporary files even when an uncaught exception occurs
+tmp.setGracefulCleanup()
+
+createImporter = (format) ->
+  Config.importFormat = format || "csv"
```

Comment:
why are we updating the `Config` object in the test?

Not sure what's it's doing here, but if you want to update the properties in it, you can clone it and alter.